.gitignore

```diff
@@ -24,3 +24,4 @@ tests/testthat/testthat-problems.rds
 renv.lock
 *.sqlite
 MergedCohortDiagnosticsData.7z
+*.csv
```

Comment:
We use csv files for settings and ddls. Please remove from the gitignore

Reply:
done